assets/js/common/Input/Password.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+
+import {
+  EOS_VISIBILITY_OFF_OUTLINED,
+  EOS_VISIBILITY_OUTLINED,
+} from 'eos-icons-react';
+import classNames from 'classnames';
+import Input from './Input';
+
+function Password({
+  className,
+  id,
+  name,
+  value,
+  placeholder = 'Password',
+  allowToggleVisibility = false,
```

Comment:
I would strip off this, I think we just want visibility to always be togglable. Your call nonetheless.

Reply:
I don't mind much, but if we want one day to also use this Password component in the login page it might be useful :eyes:

Reply:
I personally don't have a big problem with that, for example here's my Fastmail login screen that lets me toggle password visibility for the login screen :+1: it would actually be a feature to me :smile:

![immagine](https://github.com/trento-project/web/assets/20770/c0f03543-1928-49f4-ada5-d728a081928e)

@jagabomb what do you think?

Reply:
Yes, good point. Happy to make that the default behavior in case. 👍

Reply:
@nelsonkopliku This makes sense to me. Always have the password visibility toggle present.